core/Container/ContainerFactory.php

```diff
@@ -82,4 +82,16 @@ private function addEnvironmentConfig(ContainerBuilder $builder)
 
         $builder->addDefinitions($file);
     }
+
+    private function addPluginConfigs(ContainerBuilder $builder)
+    {
+        $plugins = Config::getInstance()->Plugins['Plugins'];
+
+        foreach ($plugins as $plugin) {
+            $file = Manager::getPluginsDirectory() . $plugin . '/plugin.php';
+            if (file_exists($file)) {
```

Comment:
> Would also allow to add maybe other config entries if needed. Maybe the config folder would be also useful if we allow to provide something like config-dev.php or config/environment/dev.php that will be automatically used for development etc?

Great point, I'm +1 with that idea then, I'll move again to `config/config.php`.